charter.md

```diff
@@ -0,0 +1,74 @@
+# SIG Release Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+- Production of Kubernetes releases on a reliable schedule
+- Ensure there is a consistent group of community members in place to support the release process across time
+- Provide guidance and tooling to facilitate the production of automated releases
+- Serve as a tightly integrated partner with other SIGs to empower SIGs to integrate their repositories into the release process
+
+### In scope
+
+- Ensuring quality Kubernetes releases
+  - Defining and staffing release roles to manage the resolution of release blocking criteria
+  - Defining and driving development processes (e.g. merge queues, cherrypicks) and release processes
+    (e.g. burndown meetings, cutting beta releases) with the intent of meeting the release schedule
+  - Managing the creation of release specific artifacts, including:
+    - Code branches
+    - Binary artifacts
+    - Release notes
+- Continually improving release and development processes
+  - Working closely with SIG Contributor Experience to define and build tools to facilitate release process (e.g. dashboards)
+  - Working closely with SIG Testing to determine and implement tests, automation, and labeling required for stable releases
+  - Working with downstream communities responsible for packaging Kubernetes releases
+  - Working with other SIGs to agree upon the responsibilities of their SIG with respect to the release
+  - Defining and collecting metrics related to the release in order to measure progress over each release
+  - Facilitating release retrospectives in conjunction with SIG PM
+- Collaborating with downstream communities which build artifacts from Kubernetes releases
+  - Ensuring the appropriate level of integration with publishing the build artifacts
+
+### Out of scope
+
+#### Support
+
+SIG Release itself is not responsible for end user support or creation of patches for support streams. There are support forums for end users to ask questions and report bugs, subject matter experts in other SIGs triage and address issues and when necessary mark bug fixes for inclusion in a patch release.
+
+## Roles and Organization Management
+
+This SIG adheres to the Roles and Organization Management outlined in [sig-governance] and opts-in to updates and modifications to [sig-governance].
+
+Specifically, the common guidelines (see: [sig-governance]) for continuity of membership within roles in the SIG are followed.
+
+### Deviations from [sig-governance]
+
+- SIG Release does not have top-level SIG Technical Leads; SIG Release Chairs can serve in both an operational and technical capacity
```

Comment:
I would caveat this by saying that we don't _currently_ have any technical leads in deference to subproject leads

Reply:
The intent was that we don't require Technical Leads, either now or in the future, as our Chairs are capable of executing technical duties, as well as the fact that we'll have multiple subproject chairs for specific areas. Do you feel that's captured here or disagree with that sentiment?

Reply:
Asking the SIG release chair to be "technical" is not inclusionary language. It also implies vague powers for technical arbitration that rightly belongs to sub-projects. Chairs are responsible for the organization, facilitation, and administrivia of the umbrella group. In that regard, SIG Release is not different than any other SIG WRT chairs. I would strike this language.

Reply:
Yeah, realizing now that the phrasing didn't match my intent. So stricken.